src/android/orb.yml

```diff
@@ -224,6 +224,12 @@ commands:
       - run:
           name: Check if library artifacts should be published
           command: |
+            echo "Environment:"
+            echo "> SHA1: $CIRCLE_SHA1"
+            echo "> tag?: $CIRCLE_TAG"
+            echo "> branch?: $CIRCLE_BRANCH"
+            echo "> PR?: $CIRCLE_PULL_REQUEST"
```

Comment:
When reviewing https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/73, I was confused by the fact that it didn't publish to S3.

This information would have helped make sense of the halt message.